htdocs/compta/paymentbybanktransfer/index.php

```diff
@@ -97,35 +101,34 @@
 print '</span></td></tr></table></div><br>';
 
 
-
 /*
  * Invoices waiting for withdraw
  */
-$sql = "SELECT f.ref, f.rowid, f.total_ttc, f.fk_statut, f.paye, f.type, f.datef, f.date_lim_reglement,";
-$sql .= " pfd.date_demande, pfd.amount,";
-$sql .= " s.nom as name, s.email, s.rowid as socid, s.tva_intra, s.siren as idprof1, s.siret as idprof2, s.ape as idprof3, s.idprof4, s.idprof5, s.idprof6";
-$sql .= " FROM ".MAIN_DB_PREFIX."facture_fourn as f,";
-$sql .= " ".MAIN_DB_PREFIX."societe as s";
-if (empty($user->rights->societe->client->voir) && !$socid) {
-	$sql .= ", ".MAIN_DB_PREFIX."societe_commerciaux as sc";
-}
-$sql .= ", ".MAIN_DB_PREFIX."prelevement_demande as pfd";
-$sql .= " WHERE s.rowid = f.fk_soc";
-$sql .= " AND f.entity IN (".getEntity('supplier_invoice').")";
-$sql .= " AND f.total_ttc > 0";
-if (empty($conf->global->WITHDRAWAL_ALLOW_ANY_INVOICE_STATUS)) {
-	$sql .= " AND f.fk_statut = ".FactureFournisseur::STATUS_VALIDATED;
-}
-$sql .= " AND pfd.traite = 0";
-$sql .= " AND pfd.ext_payment_id IS NULL";
-$sql .= " AND pfd.fk_facture_fourn = f.rowid";
-if (empty($user->rights->societe->client->voir) && !$socid) {
-	$sql .= " AND s.rowid = sc.fk_soc AND sc.fk_user = ".((int) $user->id);
-}
-if ($socid) {
-	$sql .= " AND f.fk_soc = ".((int) $socid);
-}
-
+// $sql = "SELECT f.ref, f.rowid, f.salary, f.paye,";
```

Comment:
You prepare new code for the salary support but you removed the code for credit transfer of invoices.
Salary should be added as a second part but must not remove the old code for supplier invoices